formatTest/typeCheckedTests/expected_output/jsx.re

```diff
@@ -95,8 +95,8 @@ module Namespace = {
 };
 
 module Optional1 = {
-  let createElement ::optional children =>
-    switch optional {
+  let createElement ::requiredOpt children =>
```

Comment:
Is my understanding correct that this is not *technically* an "optional" labeled argument right? It's just a regular labeled argument that happens to be of type `option string`.